src/czml3/properties.py

```diff
@@ -396,6 +471,34 @@ class Point(BaseCZMLObject):
     disableDepthTestDistance = attr.ib(default=None)
 
 
+@attr.s(repr=False, frozen=True, kw_only=True)
+class TileSet(BaseCZMLObject):
+    """A 3D Tiles tileset."""
+
+    show = attr.ib(default=None)
+    uri = attr.ib()
+    maximumScreenSpaceError = attr.ib(default=None)
+
+
+@attr.s(repr=False, frozen=True, kw_only=True)
+class Wall(BaseCZMLObject):
+    """A two-dimensional wall defined as a line strip and optional maximum and minimum heights.
+     It  conforms to the curvature of the globe and can be placed along the surface or at altitude."""
+
+    show = attr.ib(default=None)
+    positions = attr.ib(default=None)
```

Comment:
This appears to be required: https://github.com/AnalyticalGraphicsInc/czml-writer/wiki/Wall#properties

Reply:
```suggestion
    positions = attr.ib()
```

Reply:
Yes indeed!